PayloadPkg/Include/MeFwUpdateLib.h

```diff
@@ -0,0 +1,238 @@
+/** @file
+
+  Copyright (c) 2017 - 2019, Intel Corporation. All rights reserved.<BR>
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+
+**/
+
+
+#ifndef __ME_FW_UPDATELIB_H__
+#define __ME_FW_UPDATELIB_H__
+
+#define FPT_PARTITION_NAME_FTPR         0x52505446
+#define FPT_PARTITION_NAME_RBEP         0x50454252
+#define FPT_PARTITION_NAME_NFTP         0x5054464E
+#define FPT_PARTITION_NAME_PMCP         0x50434D50
+#define FPT_PARTITION_NAME_PCHC         0x43484350
+#define FPT_PARTITION_NAME_OEMP         0x504D454F
+#define FPT_PARTITION_NAME_ISHC         0x43485349
+#define FPT_PARTITION_NAME_IUNP         0x504E5549
+#define FPT_PARTITION_NAME_LOCL         0x4C434F4C
+#define FPT_PARTITION_NAME_WCOD         0x444f4357
+#define FPT_PARTITION_NAME_IOMP         0x504D4F49
+#define FPT_PARTITION_NAME_MGPP         0x5050474D
+#define FPT_PARTITION_NAME_TBTP         0x50544254
+#define FPT_PARTITION_NAME_DPHY         0x59485044
+
+#define MFT_PART_INFO_EXT_UPDATE_ACTION_NONE         0
+#define MFT_PART_INFO_EXT_UPDATE_ACTION_HOST_RESET   1
+#define MFT_PART_INFO_EXT_UPDATE_ACTION_CSE_RESET    2
+#define MFT_PART_INFO_EXT_UPDATE_ACTION_GLOBAL_RESET 3
+
+#define FW_UPDATE_DISABLED 0
+#define FW_UPDATE_ENABLED 1
+
+#define FWU_FW_TYPE_INVALID 0
+#define FWU_FW_TYPE_RESERVED 1
+#define FWU_FW_TYPE_SLIM 2
+#define FWU_FW_TYPE_CONSUMER 3
+#define FWU_FW_TYPE_CORPORATE 4
+
+#define FWU_PCH_SKU_INVALID 0
+#define FWU_PCH_SKU_H 1
+#define FWU_PCH_SKU_LP 2
+
+/**
+  Starting a Full FW Update from a buffer.
+
+  @param[in]  Buffer        Buffer of Update Image read from Update Image file.
+  @param[in]  BufferLength  Length of the buffer in bytes.
+  @param[in]  OemId         OEM ID to compare with OEM ID in FW (if exist). Can be NULL.
+  @param[in]  Func          A callback function that reports the progress of sending the buffer
+                            to FW (not the progress of the update itself). Can be NULL.
+                            void* Func(UINT32 BytesSentToFw, UINT32 TotalBytesToSendToFw);
+  
+  @retval SUCCESS  If Update started successfully. Error code otherwise.
+**/
+UINT32 FwuFullUpdateFromBuffer(IN  UINT8 *Buffer,
```

Comment:
pls follow coding style for all functions prototype.
UINT32
FwuFullUpdateFromBuffer(
  IN ...
)

Reply:
Done, Corrected them